test/Microsoft.Azure.Devices.Edge.Test.Common/EdgeDevice.cs

```diff
@@ -14,20 +15,50 @@ public class EdgeDevice
         readonly IotHub iotHub;
         readonly bool owned;
 
-        EdgeDevice(Device device, bool owned, IotHub iotHub)
+        readonly NestedEdgeConfig nestedEdgeConfig;
+
+        public class NestedEdgeConfig
```

Comment:
>NestedEdgeConfig [](start = 21, length = 16)

actually I'm not seeing why this class is needed, looks like the only thing that's being used out of here is the ParentDeviceId?

Reply:
Yup